src/Uno.Sdk/Sdk/Sdk.props

```diff
@@ -54,6 +54,10 @@ Copyright (C) Uno Platform Inc. All rights reserved.
 		<DisableImplicitAspNetCoreAnalyzers Condition=" '$(DisableImplicitAspNetCoreAnalyzers)' == '' ">true</DisableImplicitAspNetCoreAnalyzers>
 	</PropertyGroup>
 
+	<PropertyGroup>
+		<_RequiresILLinkPack Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'browserwasm'">true</_RequiresILLinkPack>
+	</PropertyGroup>
```

Comment:
I wonder why this doesn't happen here:

https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/9b559fc9a8b5e939fd99fa119a243e760ae3770d/src/Uno.Wasm.Bootstrap/build/Uno.Wasm.Bootstrap.targets#L52

and also if the adjustment should be made on the bootstrapper side